### PR TITLE
lifecycle: specify error when logging

### PR DESCRIPTION
**Pull request by jmontesi**
Split combined error log to show the reason.

### CODE REVIEW

File: test-network-function/lifecycle/suite.go [hunk 1]
```diff
@@ -694,8 +694,12 @@ func drainNode(node string, context *interactive.Context) {
 	test, err := tnf.NewTest(context.GetExpecter(), tester, []reel.Handler{tester}, context.GetErrorChannel())
 	gomega.Expect(err).To(gomega.BeNil())
 	result, err := test.Run()
-	if err != nil || result == tnf.ERROR {
-		log.Fatalf("Test skipped because of draining node failure - platform issue")
+	if err != nil {
+		log.Fatalf("Test skipped because of draining node failure - platform issue, err: %v", err)
```
Comment (greyerof) [inline test-network-function/lifecycle/suite.go:698]: The TC is not skipped, it results in an abrupt program exit due to log.Fatalf being used, so please remove that part and change the message to:
`log.Fatalf("Failed to drain node %s, err: %v", node, err)`

File: test-network-function/lifecycle/suite.go [hunk 2]
```diff
@@ -694,8 +694,12 @@ func drainNode(node string, context *interactive.Context) {
...
+	}
+
+	if result == tnf.ERROR {
+		log.Fatalf("Test skipped because of draining node failure - NewTest returned tnf.ERROR")
```
Comment (greyerof) [inline test-network-function/lifecycle/suite.go:702]: Same here:
`log.Fatalf("Failed to drain node %s. The tester returned tnf.ERROR", node)`

**Review by greyerof — APPROVED** [review]

File: test-network-function/lifecycle/suite.go (hunk 1 above)
Comment (jmontesi) [reply test-network-function/lifecycle/suite.go]: Done

File: test-network-function/lifecycle/suite.go (hunk 2 above)
Comment (jmontesi) [reply test-network-function/lifecycle/suite.go]: Done

**Comment by jmontesi** [comment]
To be continued in https://github.com/test-network-function/test-network-function/pull/646.

**Pull request closed without merging**